csrc/ops.h

```diff
@@ -129,6 +129,16 @@ void static_scaled_fp8_quant(torch::Tensor& out, torch::Tensor& input,
 void dynamic_scaled_fp8_quant(torch::Tensor& out, torch::Tensor& input,
                               torch::Tensor& scale);
 
+void convert_fp8(torch::Tensor& dst_data, torch::Tensor const& src_data,
+                 torch::Tensor const& scale);
+
+#ifdef USE_ROCM
+void fp8_mm(torch::Tensor& a, torch::Tensor& b, torch::Tensor& result,
```

Comment:
Since it's ROCm specific, we should call it `fp8_mm_rocm` or something similar to avoid confusion.